devicetypes/fibargroup/fibaro-co-sensor-zw5.src/fibaro-co-sensor-zw5.groovy

```diff
@@ -92,7 +92,7 @@ metadata {
 }
 
 def installed() {
-	sendEvent(name: "checkInterval", value: 86520, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	sendEvent(name: "checkInterval", value: 43632, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])

```

Comment:
I always end up doing the math to figure out how long that really is, could you do something like `12 * 60 * 60  + whatever`